CONTRIBUTING.md

```diff
@@ -98,4 +98,4 @@ For linting JavaScript, a [jshint file](https://github.com/nextcloud/news/blob/m
 When you commit your change, remember to sign off that you adhere to [DCO requirements](https://developercertificate.org/) as described by [Probot](https://probot.github.io/apps/dco/).
 
 ### Change log
-Before you create a pull request, please remember to add an entry to [CHANGELOG.md](https://github.com/nextcloud/news/blob/master/CHANGELOG.md), using the format [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
\ No newline at end of file
+Before you create a pull request, please remember to add an entry to [CHANGELOG.md](https://github.com/nextcloud/news/blob/master/CHANGELOG.md), using the format [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Comment:
```suggestion
Before you create a pull request, please remember to add an entry to [CHANGELOG.md](https://github.com/nextcloud/news/blob/HEAD/CHANGELOG.md), using the format [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Reply:
For the signoff should I use my real name or the login name for github?